javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,26 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+
+console.log(products);
+
+function renderProducts(products) {
+  for (const product of products) {
+    const productsUl = document.getElementById('productsUl');
+    const productLi = document.createElement('li');
+    const productName = document.createElement('h3');
+    const productPrice = document.createElement('p');
+    const productRating = document.createElement('p');
+
+    productsUl.appendChild(productLi);
+    productLi.appendChild(productName);
+    productLi.appendChild(productPrice);
+    productLi.appendChild(productRating);
+
+    productName.innerHTML = product.name;
+    productPrice.innerHTML = `Price: ${product.price}`;
+    productRating.innerHTML = `Rating: ${product.rating}`;
+  }
+}
+
+renderProducts(products);
```

Comment:
Hi Yuliia,
Seems good but , I had some issues running this file, I got below error, can you check it again. 

main.js:8 Uncaught ReferenceError: ul is not defined
    at renderProducts (main.js:8:9)
    at main.js:18:1

Reply:
Hi. Thank you for review. I don't have any errors. 
![image](https://user-images.githubusercontent.com/73549460/176355666-908e2999-a321-4526-bbf9-daccdaea263d.png)

Reply:
Hej Yuliia, i checked again it is working . please ignore this comment .